lib/generation/generators/layouts/pb_scaffold_gen.dart

```diff
@@ -19,35 +19,39 @@ class PBScaffoldGenerator extends PBGenerator {
     var bottomNavBar = source.getAttributeNamed(tree, 'bottomNavigationBar');
     if (source is InheritedScaffold) {
       var buffer = StringBuffer();
-      buffer.write('Scaffold(\n');
```

Comment:
I think that rather than adding a hack to interpret Scaffolds as Material widget, we should create an `inherited_material.dart` or something along those lines with its own generator. We can then place it higher than the `InheritedScaffold` (priority-wise) in the `abstract_intermediate_node_factory.dart`.

This way, we can then easily add a flag or environment variable that allows users to decide whether they want to isolate their components or generate scaffolds (when we add full support for that). Let me know if you see any downsides to this though 🤔

Reply:
That sounds good to me, let me change it right now